_notifications-plugin/api.md

```diff
@@ -0,0 +1,401 @@
+---
+layout: default
+title: API
+nav_order: 50
+has_children: false
+redirect_from:
+---
+
+# Notifications API
+
+If you want to programmatically define your notifications channels and sources for versioning and reuse, you can use the Notifications REST API to define, configure, and delete notification channels, as well as send test messages.
+
+---
+
+#### Table of contents
+1. TOC
+{:toc}
+
+---
+
+## List supported channel types
+
+Returns a list of supported channel types.
```

Comment:
The `features` API returns a list of all channel types that the Notifications plugin supports.